core/che-core-api-core/src/main/java/org/eclipse/che/api/core/util/ProcessUtil.java

```diff
@@ -42,7 +44,7 @@
    * @param stderr a consumer where stderr will be redirected
    * @throws IOException
    */
-  public static void process(Process p, LineConsumer stdout, LineConsumer stderr)
+  public static void readOutput(Process p, LineConsumer stdout, LineConsumer stderr)
```

Comment:
It isn't a good idea to refactor core util class

Reply:
I see. Will restore the previous name.